homework.js

```diff
@@ -0,0 +1,342 @@
+/*
+Exercise 1.
+Create two variables called width and height .
+Each of those variables should hold some number.
+Then, create a new variable that holds the area of a rectangle, calculated based on the width and height variables.
+Print it to the console.
+ */
+console.log("--- Exercise 1 ---");
+const rectangleWidth = 10;
+const rectangleHeight = 11;
+
+const rectangleArea = rectangleWidth * rectangleHeight;
+console.log('rectangleArea:', rectangleArea);
+
+/*
+Exercise 2.
+Create two variables called firstNumber and secondNumber.
+Assign each of them with a different number.
+Write a program that swaps both values.
+ */
+console.log("--- Exercise 2 ---");
+let firstNumber = 10;
+let secondNumber = 11;
+const temporaryNumber = firstNumber;
+
+firstNumber = secondNumber;
+secondNumber = temporaryNumber;
+
+/*
+Exercise 3.
+Create variables that hold grades from three different classes:
+Physics
+Math
+English
+Calculate the average of the grades.
+If the average equals 4 out of 5 or higher, print "Good job!".
+Otherwise, print "Try harder!"
+ */
+console.log("--- Exercise 3 ---");
+const physicsGrade = 3;
+const mathGrade = 4;
+const englishGrade = 4;
+
+const gradeAverage = (physicsGrade * mathGrade + englishGrade) / 3;
+if (gradeAverage >= 4 && gradeAverage <= 6) {
+    console.log('Good job!')
+} else if (gradeAverage < 4 && gradeAverage >= 1) {
+    console.log('Try harder!')
+} else {
+    console.log('Wrong grades!')
+}
+
+/*
+Exercise 4.
+Fix the code.
+ */
+console.log("--- Exercise 4 ---");
+const currentAge = 30;
+const retirenmentAgeInPoland = 67;
+let yearsLeftToRetirenment = retirenmentAgeInPoland;
+
+yearsLeftToRetirenment = retirenmentAgeInPoland - currentAge;
+
+console.log(yearsLeftToRetirenment);
+
+/*
+Exercise 5.
+Fix the code (2 mistakes).
+COMMENT: 4? VATRateInPoland - variable name and type (wasn't const), taxToPay type (was const), monthlyIncome type (wasn't const)
+ */
+console.log("--- Exercise 5 ---");
+const isPayingVAT = true;
+const vatRateInPoland = 0.23;
+const incomeTaxRate = 0.12;
+const monthlyIncome = 3500;
+let taxToPay = 0;
+
+if (isPayingVAT) {
+    taxToPay = monthlyIncome * vatRateInPoland;
+}
+
+taxToPay = taxToPay + monthlyIncome * incomeTaxRate;
+
+console.log('tax to pay: ' + taxToPay);
+
+/*
+Exercise 6.
+Declare a variable called isSunny and assign a boolean value to it.
+Create an if-else statement that checks if it's sunny.
+If it is, print "Don't forget your sunglasses!".
+If it's not, print "You might need an umbrella."
+ */
+console.log("--- Exercise 6 ---");
+const isSunny = true;
+if (isSunny === true) {
```

Comment:
remember that syntax for `if` statement is as follows:
```
if(<expression that evaluates to truthy>) {
 // code
}
```
so you can do 
```suggestion
if (isSunny) {
```
Because `isSunny` is already of boolean type, no need to create a new expression that checks if `true === true` or `false === true`.

Reply:
Right, it is good idea to simplify that :)
Resolved